### PR TITLE
Quickstart overview update

### PR DESCRIPTION
**Pull request by StevenTCramer**
Updated screen shots to reflect latest versions of Identity Server.  Cleaned up small wording issues. Updated to latest template code.  Like leaving in the `if (env.IsDevelopment())`and now UseApplicationInsights is in the default.

I removed the line:
> "In fact, when you try it, you should see an exception at startup time stating that services are missing."  

because the exception doesn't actually say that.

built the docs using "read the docs". Can be reviewed at the following:
http://identityserver4test.readthedocs.io/en/latest/quickstarts/0_overview.html

### CODE REVIEW

File: docs/quickstarts/0_overview.rst [hunk 1]
```diff
@@ -107,6 +111,7 @@ we use the following configuration for the IdentityServer host in the quickstart
                 .UseContentRoot(Directory.GetCurrentDirectory())
                 .UseIISIntegration()
                 .UseStartup<Startup>()
+                .UseApplicationInsights()
```
Comment (brockallen) [inline docs/quickstarts/0_overview.rst]: please remove this

**Comment by brockallen** [comment]
looks good, other than the app insights -- remove that one line, please. pizza pizza!

File: docs/quickstarts/0_overview.rst (hunk 1 above)
Comment (StevenTCramer) [inline docs/quickstarts/0_overview.rst]: Can do.

Comment (StevenTCramer) [inline docs/quickstarts/0_overview.rst]: removed :)

**Comment by StevenTCramer** [comment]
Updated.  And thanks for the Pizza Pizza :)

**Comment by leastprivilege** [comment]
thanks!

**Pull request merged**